coretex/cli/modules/node.py

```diff
@@ -307,7 +375,10 @@ def initializeNodeConfiguration() -> None:
     config = loadConfig()
 
     if isNodeConfigured(config):
-        return
+        if isConfigurationValid(config):
```

Comment:
`if firstCondition and secondCondition and thirdCondition` is equivalent to `if firstCondition: if secondCondition: if thirdCondition` functionaly, but the former is much easier to read as it does not have unnecessary nesting. Refactor this whole if-else branching as there is no need for the else block to exist at all.